pkg/cli/cliflags/flags.go

```diff
@@ -1652,6 +1652,18 @@ For large clusters, this can dramatically increase debug zip size/file count.
 `,
 	}
 
+	ZipIncludeGoroutineStacks = FlagInfo{
+		Name: "include-goroutine-stacks",
+		Description: `
+Fetch stack traces for all goroutines running on each targeted node in nodes/*/stacks.txt
+and nodes/*/stacks_with_labels.txt files. Note that fetching stack traces for all goroutines is
+a "stop-the-world" operation, which can momentarily have negative impacts on SQL service 
+latency. Note that any periodic goroutine dumps previously taken on the node will still be 
+included in nodes/*/goroutines/*.txt.gz, as these would have already been generated and don't 
+require any stop-the-world operations.
```

Comment:
what makes us say that the goroutine dumps previously taken and sitting on disk  `don't require any stop-the-world operations.` ? those call the same `runtime.Stack(..., true)` don't they?

Reply:
This is meant to say that they don't trigger any "stop the world" operations during the debug zip operation. We just pull the files.

The goroutine dumper uses `pprof`, not `runtime.Stack(..., true)`: https://github.com/cockroachdb/cockroach/blob/4640c585e7098c53ccc69f6096e8087b6d6007fd/pkg/server/goroutinedumper/goroutinedumper.go#L170

Reply:
Ah, I see. Perhaps this should say "don't require any _additional_ stop-the-world operations to be collected from disk".

> The goroutine dumper uses pprof, not `runtime.Stack`

https://github.com/golang/go/blob/d80f3966b8b5b2a6e606b416fe80dffeb810ab68/src/runtime/pprof/pprof.go#L703

Reply:
> The goroutine dumper uses pprof, not runtime.Stack

> > https://github.com/golang/go/blob/d80f3966b8b5b2a6e606b416fe80dffeb810ab68/src/runtime/pprof/pprof.go#L703

Lol, well there ya go! 

Pushed a patch to fix the phrasing, as requested.